gnuradio-runtime/lib/top_block_impl.cc

```diff
@@ -50,10 +50,9 @@ make_scheduler(flat_flowgraph_sptr ffg, int max_noutput_items, bool catch_except
         if (!v)
             factory = scheduler_table[0].f; // use default
         else {
-            for (size_t i = 0; i < sizeof(scheduler_table) / sizeof(scheduler_table[0]);
-                 i++) {
-                if (strcmp(v, scheduler_table[i].name) == 0) {
-                    factory = scheduler_table[i].f;
+            for (auto& i : scheduler_table) {
```

Comment:
Not a great name here either. But more generally: Many (most, probably) of these should live with getting a `const` for readability.

Reply:
yeah the heuristic with which the `const`ness is determined is really rather ... conservative. At least it doesn't do any copies as far as I looked through it.